libraries/ESP8266HTTPUpdateServer/src/ESP8266HTTPUpdateServer.cpp

```diff
@@ -8,12 +8,19 @@
 
 const char* ESP8266HTTPUpdateServer::_serverIndex =
 R"(<html><body><form method='POST' action='' enctype='multipart/form-data'>
+		  <input type='hidden' name='cmd' value='0'>
                   <input type='file' name='update'>
-                  <input type='submit' value='Update'>
+                  <input type='submit' value='Update Flash'>
+               </form>
+	       <form method='POST' action='' enctype='multipart/form-data'> 
+		  <input type='hidden' name='cmd' value='100'>
+                  <input type='file' name='update'>
+                  <input type='submit' value='Update Spiffs'>
                </form>
          </body></html>)";
 const char* ESP8266HTTPUpdateServer::_failedResponse = R"(Update Failed!)";
 const char* ESP8266HTTPUpdateServer::_successResponse = "<META http-equiv=\"refresh\" content=\"15;URL=\">Update Success! Rebooting...";
+int _command;
```

Comment:
Could you please make the `int _command` a class member in `ESP8266HTTPUpdateServer`?

Reply:
Hi @PA4WD, do you need help with that?
This PR will be a great improvement to this feature!